releasenotes/config.yaml

```
@@ -1,3 +1,49 @@
 ---
 encoding: utf8
 default_branch: main
+collapse_pre_releases: true
+sections:
+    - [features, New Features]
+    - [features_algorithms, Algorithms Features]
+    - [features_circuits, Circuits Features]
+    - [features_primitives, Primitives Features]
+    - [features_providers, Providers Features]
+    - [features_pulse, Pulse Features]
+    - [features_qasm, OpenQASM Features]
+    - [features_qpy, QPY Features]]
```

This is causing the docs and lint failure I think:

```suggestion
    - [features_qpy, QPY Features]
```

---

Yes, already fixed but not pushed yet. Fingers crossed.